templates/paragraphs/paragraph--oe-text-feature-media.html.twig

```diff
@@ -6,9 +6,26 @@
  * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
  */
 #}
+
+{#
+  Unfortunately even when the field is empty the variable content.* still contains information (such as the weight)
+  so we can't pass it to the pattern directly.
+ #}
+{% if not (paragraph.field_oe_title.isEmpty == true) and content.field_oe_title is defined %}
+  {% set _title = content.field_oe_title %}
+{% endif %}
+
+{% if not (paragraph.field_oe_plain_text_long.isEmpty == true) and content.field_oe_plain_text_long is defined %}
+  {% set _caption = content.field_oe_plain_text_long %}
+{% endif %}
+
+{% if not (paragraph.field_oe_text_long.isEmpty == true) and content.field_oe_text_long is defined %}
+  {% set _text = content.field_oe_text_long %}
+{% endif %}
+
 {{ pattern('text_featured_media', {
-    'title': content.field_oe_title,
+    'title': _title|default(''),
     'image': image,
-    'caption': content.field_oe_plain_text_long,
-    'text': content.field_oe_text_long,
+    'caption': _caption|default(''),
+    'text': _text|default(''),
```

Comment:
@imanoleguskiza would just rendering it work? 
```twig
{{ pattern('text_featured_media', {
    'title': content.field_oe_title,
    'image': image,
    'caption': content.field_oe_plain_text_long|render,
    'text': content.field_oe_text_long,
...
```
Unless we have HTML in that, then try with a
``` 
'caption': content.field_oe_plain_text_long|render|raw,`
```

If not then this would be enough I guess (field names are might not be correct):

```twig
{{ pattern('text_featured_media', {
    'title': not paragraph.field_oe_title.isEmpty ? content.field_oe_title,
    'image': not paragraph.field_oe_image.isEmpty ? image,
    'caption': not paragraph.field_oe_plain_text_long.isEmpty ? content.field_oe_plain_text_long,
    'text': not paragraph.field_oe_text_long.isEmpty ? content.field_oe_text_long,
}) }}
```

Reply:
Probably this is the best we have as, if we pre-render, we will get theme suggestion info in the rendered html when twig debug is on, which will confuse developers:

```
{{ pattern('text_featured_media', {
    'title': not paragraph.field_oe_title.isEmpty ? content.field_oe_title,
    'image': not paragraph.field_oe_image.isEmpty ? image,
    'caption': not paragraph.field_oe_plain_text_long.isEmpty ? content.field_oe_plain_text_long,
    'text': not paragraph.field_oe_text_long.isEmpty ? content.field_oe_text_long,
}) }}
```